includes/class-recaptcha.php

```diff
@@ -7,22 +7,33 @@
 
 namespace Newspack;
 
+use Error;
+
 defined( 'ABSPATH' ) || exit;
 
 /**
  * Class for reCAPTCHA integration.
  */
 final class Recaptcha {
-	const SCRIPT_HANDLE  = 'newspack-recaptcha';
-	const OPTIONS_PREFIX = 'newspack_recaptcha_';
+	const SCRIPT_HANDLE      = 'newspack-recaptcha';
+	const SCRIPT_HANDLE_API  = 'newspack-recaptcha-api';
+	const OPTIONS_PREFIX     = 'newspack_recaptcha_';
+	const SUPPORTED_VERSIONS = [ 'v3', 'v2_checkbox', 'v2_invisible' ];
```

Comment:
Is `v2_checkbox` intended here? It's not available in the Connections wizard settings.

Reply:
Yeah, since we originally went through the trouble of adding an implementation for the v2 Checkbox flavor, I left the infrastructure in place so it'd be easy to reenable in the future if we ever want to. All we'd have to do is readd it as an option in the Connections UI.

Reply:
4f51b03c removes the string from the `SUPPORT_VERSIONS` constant, but adds a note in the relevant areas of the code where we could add it to reenable support for the Checkbox flavor.